OpenRA.Game/Network/Connection.cs

```diff
@@ -167,12 +167,12 @@ void NetworkConnectionReceive(object networkStreamObject)
 			{
 				var networkStream = (NetworkStream)networkStreamObject;
 				var reader = new BinaryReader(networkStream);
-				var serverProtocol = reader.ReadInt32();
+				var handshakeProtocol = reader.ReadInt32();
 
-				if (serverProtocol != ProtocolVersion.Version)
+				if (handshakeProtocol != ProtocolVersion.Handshake)
```

Comment:
The handshaking process is a different version of the protocol?

Isn't the protocol numbered?